migrations/dev-20210621-add-support-for-update-emojis.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    await queryInterface.sequelize.query('ALTER TABLE "CommentReactions" RENAME TO "EmojiReactions";').then(() => {
+      queryInterface.sequelize.query(
+        'ALTER TABLE "EmojiReactions" RENAME CONSTRAINT "CommentReactions_CommentId_fkey" TO "EmojiReactions_CommentId_fkey";',
+      );
+      queryInterface.sequelize.query(
+        'ALTER TABLE "EmojiReactions" RENAME CONSTRAINT "CommentReactions_FromCollectiveId_fkey" TO "EmojiReactions_FromCollectiveId_fkey";',
+      );
+      queryInterface.sequelize.query(
+        'ALTER TABLE "EmojiReactions" RENAME CONSTRAINT "CommentReactions_UserId_fkey" TO "EmojiReactions_UserId_fkey";',
+      );
+      queryInterface.sequelize.query('ALTER INDEX "CommentReactions_pkey" RENAME TO "EmojiReactions_pkey";');
```

Comment:
@Betree : My thinking is that for these renames we don't care if they run asynchronously or not. 🤔 

I put them in a `then` purely for clarity and organization. So these renames all correspond to the `CommentReactions` table rename. So I thought it would be clearer for the reader to bundle them together in a `then`. But let me know if you think otherwise. 😄 🤔

Reply:
I understand, the intent makes sense. I think it's better to always await on promises, but since this is a migration that should be fine.

Reply:
I'll leave it as it is unless you like me to change it. I have no strong opinion either way on this. 😄